app/views/not_found.scala.html

```diff
@@ -36,12 +36,8 @@
             location = appConfig.financialsHomepage,
             linkClass = "govuk-link govuk-link--no-visited-state",
             preLinkMessage = Some("cf.error.not-found.message.go-to-home-page-text1"),
-            postLinkMessage = Some("cf.error.not-found.message.go-to-home-page-text2")
+            linkSentence = true
```

Comment:
Is this variable defined in the component?

Reply:
yes this is defined and has a default value of false